trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+# This class is a decorator that modifies the behavior of the correct_name method
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice(0, 10)
```

Comment:
- [ ] Great job working here but I think you forgot to add a condition here as mentioned in the requirements `. If it's longer it should trim the word.` So kindly add this condition here si if the word longer than 10 then trim ✨🔥✅


![image](https://github.com/rudicarrilloypr/school-library-opp/assets/58553711/1d94fb4c-3cc8-415d-8cea-46a9e8bab035)